MotorControl.c

```diff
@@ -5,20 +5,44 @@
  *      Author: brianvuong
  */
 
+#include <xdc/cfg/global.h>
+
 #include "MotorControl.h"
 #include "PWM.h"
 
 #include <string.h>
 
+/* TI-RTOS Header files */
+#include <ti/drivers/GPIO.h>
+#include "board.h"
+
+// TODO!
+#include <msp430.h>
```

Comment:
Refactor as part of issue [24](https://github.com/thaletterb/BLDC_ESC/issues/24)
* Remove TA0CCRx values